docs/guides/security/openssl-guide.md

```diff
@@ -0,0 +1,25 @@
+# OpenSSL Guide
+
+This document provides information about supported OpenSSL versions and
+security details that you need to consider.
+
+OpenSSL is a package dependency as infrap4d uses the library for gRPC.
+
+## End of Life for OpenSSL 1.1.1
+
+OpenSSL 1.1.1 has reached End of Life (EOL) in September 2023. 
+
+It is highly recommended that you upgrade OpenSSL from 1.1.1x to OpenSSL 3.x.
+The official migration guide is [available here](https://www.openssl.org/docs/man3.0/man7/migration_guide.html).
```

Comment:
Integral links are better than ones that call attention to themselves (e.g., by saying "here").

```suggestion
See the [official migration guide](https://www.openssl.org/docs/man3.0/man7/migration_guide.html)
for more information.
```